### PR TITLE
Redis Dump.rdb snapshotting error detection with alerting and logging.

### PR DESCRIPTION
**Pull request by SMYALTAMASH**

### CODE REVIEW

**Comment by SMYALTAMASH** [comment]
@SanthoshVasabhaktula please review this.

File: ansible/roles/redis-dump-monitoring/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+redis_monitoring_user: analytics
+redis_monitoring_home: "/home/{{redis_monitoring_user}}"
+redis_home: "{{redis_monitoring_home}}/redis-stable"
+redis_dump_rdb_time_in_seconds: 900
+redis_dump_rdb_keys_to_save_state: 1
+redis_dump_rdb_log_file_location: "/var/log/redisdumprdbstate.log"
+redis_dump_rdb_log_file_log_rotate_backup: 14
+redis_dump_rdb_mail_sender: "monitorning@sunbirded.org"
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/redis-dump-monitoring/defaults/main.yml]: @SMYALTAMASH - Is the spelling of the mail sender correct? check the monitoring spelling

Comment (SMYALTAMASH) [reply ansible/roles/redis-dump-monitoring/defaults/main.yml]: I've  created a new variable to identify alerts specific to redis dump monitoring @SanthoshVasabhaktula and sender email name can be anything, do you want me to use the same name as existing monitoring sender default name?

Comment (SMYALTAMASH) [reply ansible/roles/redis-dump-monitoring/defaults/main.yml]: Monitoring Spelling is updated.

Comment (SMYALTAMASH) [reply ansible/roles/redis-dump-monitoring/defaults/main.yml]: > @SMYALTAMASH - Is the spelling of the mail sender correct? check the monitoring spelling



Monitoring Spelling is updated @SanthoshVasabhaktula .

**Pull request merged**